rubygem-hammer_cli_foreman_tasks/rubygem-hammer_cli_foreman_tasks.spec

```diff
@@ -8,7 +8,7 @@
 
 Summary: Foreman CLI plugin for showing task information for resources and users
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.0.8
+Version: 0.0.9
 Release: 1%{?dist}
```

Comment:
You'll need to add %{?foremandist} on rpm/develop and also here if you want to rebuild this on rpm/1.10.  Because the NEVR is identical, Koji won't permit the same build again - and since the develop version is incompatible (SCL changes), it needs a different release.